src/backend/XmlHttpRequest.ts

```diff
@@ -69,11 +75,12 @@ export default class XmlHttpRequest extends BaseFileSystem implements FileSystem
     if (typeof(opts.index) === "string") {
       XmlHttpRequest.FromURL(opts.index, cb, opts.baseUrl, false);
     } else {
-      cb(null, new XmlHttpRequest(opts.index, opts.baseUrl, false));
+      cb(null, new XmlHttpRequest(opts.index, opts.baseUrl, false, opts.use));
```

Comment:
You'll need to surround this with a try / catch since the constructor throws an exception now.

```typescript
let xhr: XmlHttpRequest;
try {
  xhr = new XmlHttpRequest(opts.index, opts.baseUrl, false, opts.use);
} catch (e) {
  return cb(e);
}
cb(null, xhr);
```

Note that surrounding the `cb` in the `try` / `catch` might accidentally catch & swallow errors thrown by user code. (I'm sure I'm forgetting to follow this advice elsewhere in the codebase, but then that's my error!)